BrCredentialCreatorWizard.vue

```diff
@@ -117,7 +119,70 @@ export default {
       return this.steps.length - 1;
     },
     steps() {
-      return [this.welcomeStep, ...this.flow, this.reviewStep];
+      const {welcomeStep, reviewStep} = this;
+
+      const steps = [];
+
+      // only add welcome step if available
+      if(welcomeStep !== null) {
+        steps.push(welcomeStep);
+      }
+
+      // add setup steps
+      if(this.setup.length > 0) {
+        steps.push(...this.setup);
+      }
+
+      // add main flow and review
+      steps.push(...this.flow);
+      steps.push(reviewStep);
```

Comment:
Why do we not have a guard checking for the existence of `reviewStep`? The computed property has similar logic as `welcomeStep()`.

Reply:
Because the `reviewStep` is not optional.

Reply:
Okay, we will need to add the `required: false` to the prop definition. This will cause an error or warning (don't remember which one) to log to the console when using Vue's dev version. I believe nothing happens with the current distribution we are using, I think it's the production version.

Reply:
@gannan08 -- I thought `required` defaulted to `false`. I tried to find this in the Vue docs but couldn't ... do you have a link?